pyproject.toml

```diff
@@ -69,7 +69,7 @@ Issues = "https://github.com/glass-dev/glass/issues"
 
 [tool.coverage]
 report = {exclude_also = [
-    "if TYPE_CHECKING:",
+    "if typing.TYPE_CHECKING:",
```

Comment:
This is important, we might potentially want to add both in case someone doesn't use the format this PR is trying to enforce